app/views/jobseekers/profile/index.html.slim

```diff
@@ -0,0 +1,12 @@
+- content_for :page_title_prefix, t(".page_title")
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    h1.govuk-heading-xl = t(".page_title")
+    p = t(".page_description")
+
+    - @sections.each do |section|
+        = tag.h2(section[:title], class: "govuk-heading-m govuk-!-font-size-27 govuk-!-padding-top-6")
```

Comment:
No need to use the tag helper here I think. Could do:

```
h2.govuk-heading-m class="govuk-!-font-size-27 govuk-!-padding-top-6"
  = section[:title]
```

Reply:
Is what we do elsewhere

Reply:
Nice!